src/main/java/hw_1/Task_3.java

```diff
@@ -0,0 +1,27 @@
+package hw_1;
+
+public class Task_3 {
+    public static void main(String[] args)
+    {
```

Comment:
this symbol "{" should be after main(String[] args). So, "public static void main(String[] args) {"

Reply:
fixed